ui_framework/src/components/flex/flex_grid.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const gutterSizeToClassNameMap = {
+  none: '',
+  small: 'kuiFlexGrid--gutterSmall',
+  medium: 'kuiFlexGrid--gutterMedium',
+  large: 'kuiFlexGrid--gutterLarge',
+  extraLarge: 'kuiFlexGrid--gutterExtraLarge',
```

Comment:
So I think I was following your own patterns here. You seemed to prefer longform in the selector names themselves.